core/src/main/java/hudson/PluginManager.java

```diff
@@ -511,6 +529,26 @@ protected boolean filter(Method e) {
         LOGGER.info("Plugin " + p.getShortName()+":"+p.getVersion() + " dynamically installed");
     }
 
+    private void onActivate(PluginWrapper pluginWrapper) {
```

Comment:
By convention, such methods are named `fireX` and are declared in the listener class.